src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -284,4 +291,24 @@ public static bool IsGet(this HttpContext context)
     {
         return string.Equals(context.Request.Method, "get", StringComparison.OrdinalIgnoreCase);
     }
+
+    public static async Task ConfigureRedis(WebApplicationBuilder builder, bool isDevelopment, string? managedIdentityId)
+    {
+        var redisConfig = ConfigurationOptions.Parse(
+            builder.Configuration.GetConnectionString("redis") ?? throw new Exception("Missing Redis connection string."));
+
+        // Local redis instance should not need authentication
+        if (!isDevelopment)
+        {
+            var azureOptions = new Microsoft.Azure.StackExchangeRedis.AzureCacheOptions();
```

Comment:
`using Microsoft.Azure.StackExchangeRedis`
```suggestion
            AzureCacheOptions azureOptions = new();
```